src/main/java/com/laserfiche/repository/api/clients/EntriesClient.java

```diff
@@ -129,9 +129,9 @@ public interface EntriesClient {
      * - Get information about the edoc content of an entry, without downloading the edoc in its entirety. - Provide an entry ID, and get back the Content-Type and Content-Length in the response headers. - This route does not provide a way to download the actual edoc. Instead, it just gives metadata information about the edoc associated with the entry.
      * @param repoId The requested repository ID. (required)
      * @param entryId The requested document ID. (required)
-     * @return Call&lt;Void&gt;
+     * @return GetDocumentContentTypeResult
```

Comment:
It looks like the other apis set the return type to Call<Type> or CompletableFuture<Type>. Does this need to be surrounded by something too? 

And should the other apis be changed to CompletableFuture return type instead of Call? I saw LinkDefinition APIs seem to return CompletableFuture  instead of Call

Reply:
Yes, you are right. I will update the code.

Reply:
Fixed, thanks!